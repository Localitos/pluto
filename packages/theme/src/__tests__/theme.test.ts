import keys from "lodash/keys";
import { DefaultTheme } from "../";

describe("Default theme", () => {
  it("should match the snapshot", () => {
    expect(keys(DefaultTheme).sort()).toMatchSnapshot();
  });
});
