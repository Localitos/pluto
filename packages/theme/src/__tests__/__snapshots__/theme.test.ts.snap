// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default theme should match the snapshot 1`] = `
Array [
  "borderWidths",
  "breakpoints",
  "colors",
  "fontSizes",
  "fontWeights",
  "fonts",
  "lineHeights",
  "radii",
  "shadows",
  "space",
  "zIndices",
]
`;
